src/components/views/rooms/BasicMessageComposer.js

```diff
@@ -444,6 +444,8 @@ export default class BasicMessageEditor extends React.Component {
             } else if (event.key === Key.TAB) {
                 this._tabCompleteName();
                 handled = true;
+            } else if (event.key === Key.BACKSPACE) {
```

Comment:
Nice, thanks for this! Could you add `DELETE` as well please?

Reply:
(Which will need to be added to Keyboard.ts: `DELETE: "Delete")

Reply:
Oh! Ok thanks for the review @bwindels I'll make those changes asap.